src/testrel.jl

```diff
@@ -420,7 +420,12 @@ function _test_rel_steps(;
             @testset TestRelTestSet nested=nested "$name" begin
                 create_test_database(schema, clone_db)
                 for (index, step) in enumerate(steps)
-                    _test_rel_step(index, step, schema, test_engine, name, length(steps))
+                    inner_ts = _test_rel_step(index, step, schema, test_engine, name, length(steps))
+                    # short circuit if something errored
+                    if anyerror(inner_ts) && index < length(steps)
+                        @error "Error running $name - not executing further steps"
```

Comment:
I'm not against changing test behaviour to stopping if errors are encountered, but the error message could be stronger, something like "Terminal error running...", to make it clear that further testing is pointless.